process/transaction/shardProcess.go

```diff
@@ -353,32 +353,27 @@ func (txProc *txProcessor) processTxFee(
 	}
 
 	moveBalanceFee := txProc.economicsFee.ComputeMoveBalanceFee(tx)
+	totalCost := txProc.economicsFee.ComputeTxFee(tx)
+	if !txProc.flagPenalizedTooMuchGas.IsSet() {
```

Comment:
I think this should be txProc.flagMetaProtection. Otherwise the sync will stop at epoch 132

Reply:
it is the same in both cases. total cost in computeTxFee is gaslimit * gasPrice

Reply:
Added test, works as expected :+1: